snare/utils/logger.py

```diff
@@ -48,6 +48,6 @@ def create_clone_logger(err_filename, logger_name):
         )
         # ERROR log to 'clone.err'
         error_log_handler = logging.handlers.RotatingFileHandler(err_filename, encoding='utf-8')
-        error_log_handler.setLevel(logging.ERROR)
+        error_log_handler.setLevel(logging.DEBUG)
```

Comment:
please change the variable names and file name, because now it's ambiguous, it's not only errors anymore

Reply:
Done.